envoy-control-core/src/test/kotlin/pl/allegro/tech/servicemesh/envoycontrol/groups/TestNodeFactory.kt

```diff
@@ -1,3 +1,4 @@
+@file:Suppress("MatchingDeclarationName")
```

Comment:
What does it check? Why do we need to disable it?

Reply:
I added OutgoingDependenciesProtoScope class to improve tests. klint/detekt want to rename file because it contains only one class.